src/test/java/teammates/test/pageobjects/InstructorCourseStudentDetailsEditPage.java

```diff
@@ -73,7 +73,7 @@ public InstructorCourseStudentDetailsEditPage submitUnsuccessfully(String studen
         if (comments != null) {
             fillTextBox(commentsTextbox, comments);
         }
-        submitButton.click();
+        clickAndConfirm(submitButton);
```

Comment:
Is there a test case where the team name isn't modified? If there isn't, can you add one?